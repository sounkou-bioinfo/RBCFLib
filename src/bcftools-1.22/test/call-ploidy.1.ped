Fam1	smpl1	0	0	1
Fam1	smpl2	0	0	2
