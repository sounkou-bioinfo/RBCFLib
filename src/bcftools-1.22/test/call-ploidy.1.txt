chr1 1 16750 1 0
chr2 1 16750 1 1
* * * 1 2
chr1 1 16750 2 2
chr2 1 16750 2 1
* * * 2 0
