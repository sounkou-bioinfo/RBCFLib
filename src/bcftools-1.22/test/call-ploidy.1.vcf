##fileformat=VCFv4.2
##reference=file://ref.fa
##contig=<ID=chr1>
##contig=<ID=chr2>
##contig=<ID=chr3>
##INFO=<ID=I16,Number=16,Type=Float,Description="Auxiliary tag used for calling, see description of bcf_callret1_t in bam2bcf.h">
##INFO=<ID=QS,Number=R,Type=Float,Description="Auxiliary tag used for calling">
##FORMAT=<ID=PL,Number=G,Type=Integer,Description="List of Phred-scaled genotype likelihoods">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	smpl1	smpl2
chr1	621	.	A	C,<*>	0	.	I16=11,0,1,0,407,15059,37,1369,440,17600,60,3600,252,6032,25,625;QS=8,1,0	PL	0,3,37,3,37,37	0,3,37,3,37,37
chr2	621	.	A	C,<*>	0	.	I16=11,0,1,0,407,15059,37,1369,440,17600,60,3600,252,6032,25,625;QS=8,1,0	PL	0,3,37,3,37,37	0,3,37,3,37,37
chr3	621	.	A	C,<*>	0	.	I16=11,0,1,0,407,15059,37,1369,440,17600,60,3600,252,6032,25,625;QS=8,1,0	PL	0,3,37,3,37,37	0,3,37,3,37,37
