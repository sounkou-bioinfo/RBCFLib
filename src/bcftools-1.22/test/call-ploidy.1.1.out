##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##reference=file://ref.fa
##contig=<ID=chr1>
##contig=<ID=chr2>
##contig=<ID=chr3>
##FORMAT=<ID=PL,Number=G,Type=Integer,Description="List of Phred-scaled genotype likelihoods">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes for each ALT allele, in the same order as listed">
##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
##INFO=<ID=DP4,Number=4,Type=Integer,Description="Number of high-quality ref-forward , ref-reverse, alt-forward and alt-reverse bases">
##INFO=<ID=MQ,Number=1,Type=Integer,Description="Average mapping quality">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	smpl1	smpl2
chr1	621	.	A	.	24.4632	.	AN=2;DP4=11,0,1,0;MQ=41	GT	.	0/0
chr2	621	.	A	.	27.9838	.	AN=2;DP4=11,0,1,0;MQ=41	GT	0	0
chr3	621	.	A	.	24.4632	.	AN=2;DP4=11,0,1,0;MQ=41	GT	0/0	.
